wurst/systems/crafting/WorkshopQuickMake.wurst

```diff
@@ -50,6 +14,15 @@ init
     ..add(ITEM_TINDER)
     ..add(ITEM_TINDER)
 
+    new QuickMake(SPELL_QM_ENSNARE_TRAP)
+    ..add(ITEM_TINDER)
+    ..add(asList(ITEM_BONE, ITEM_STICK))
+    ..add(ITEM_STICK)
+
+    new QuickMake(SPELL_QM_MANA_CRYSTAL)
```

Comment:
You can craft mana crystal now?

Reply:
yeah people said they wanted to be able to be able to craft some base components and i thought mana crystal was a good starting place